docs/sources/mimir/operators-guide/deploy-grafana-mimir/jsonnet/migrate-from-microservices-to-read-write.md

```diff
@@ -0,0 +1,311 @@
+---
+description:
+  Learn how to migrate from a cluster deployed with as microservices to
+  a cluster in read-write mode.
+menuTitle: Migrate from microservices to read-write mode
+title: Migrate from microservices to read-write mode without downtime
+weight: 40
+---
+
+# Migrate from microservices to read-write mode without downtime
+
+> **Warning:**
+> Read-Write mode is considered experimental, as is migrating between the modes.
+
+At a high level the basic process of steps to complete is:
+
+1. Deploy read-write components along side microservices (they join the same ring).
```

Comment:
```suggestion
1. Deploy read-write components alongside microservices; they join the same ring.
```
Ambiguous "they": "they" == "components", or "they" == "microservices"?

Reply:
Will update to "they both"